src/pages/teams.astro

```diff
@@ -0,0 +1,312 @@
+            Avoid the hassle and vulnerability of sharing secrets through other channels
+            and copy-pasting them to Livebook.</span
+          >
+        </div>
+        <div>
+          <div
+            class="max-w-4xl border border-gray-500 rounded-t-3xl text-center px-4 sm:px-12 py-12"
+          >
+            <div class="m-auto max-w-2xl flex flex-col items-center">
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="flag-line" class="text-xl" />
+                <span class="mt-1 text-xs font-medium tracking-[0.2em]"
+                  >ROADMAP</span
+                >
+              </div>
+              <h2 class="text-gray-50 text-2xl leading-tight mt-2">
+                Shape the future <br class="hidden sm:block" /> of Livebook Teams
+              </h2>
+              <span class="lg:w-[90%] text-gray-200 mt-6"
+                >Livebook Teams is still in development. We want to get feedback
+                from beta users and understand their use cases so that we can
+                expand and improve the product.</span
+              >
+              <a class="button button-blue mt-6" href="https://docs.google.com/forms/d/e/1FAIpQLScDfvUqT4f_s95dqNGyoXwVMD_Vl059jT6r5MPgXB99XVMCuw/viewform">
+                <span>Join the beta program</span>
+              </a>
+            </div>
+          </div>
+          <div
+            class="border-r border-b border-l border-gray-500 rounded-b-3xl pt-10 pb-16"
+          >
+            <span
+              class="mt-1 text-xs font-medium tracking-widest sm:tracking-[0.2em] text-gray-400"
+              >SOME OF THE FEATURES WE’RE CONSIDERING</span
+            >
+            <div
+              class="m-auto max-w-2xl grid grid-cols-2 sm:grid-cols-4 gap-y-5 mt-7 sm:mt-5"
+            >
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="user-add-line" class="text-xl text-blue-600" />
+                <span class="mt-1 text-sm text-gray-200">Single Sign On</span>
+              </div>
+              <div class="flex flex-col text-gray-400">
+                <Icon
+                  icon="remote-control-2-line"
+                  class="text-xl text-brand-pink"
+                />
+                <span class="mt-1 text-sm text-gray-200">Access Controls</span>
+              </div>
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="calendar-2-line" class="text-xl text-green-500" />
+                <span class="mt-1 text-sm text-gray-200">Scheduling</span>
+              </div>
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="file-list-2-line" class="text-xl text-red-600" />
+                <span class="mt-1 text-sm text-gray-200">Audit Log</span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  {
+    toggle && (
```

Comment:
We can just comment out those blocks if we want to only add them in the future, or is there a reason for the toggle?

Reply:
No other reason.

I thought that if I commented them out, the source code would still be generated inside the HTML.

If not, 👍 for commenting.

Reply:
Ah ok, that's a fair point. So we can keep it, but I would rename to something like `future`, since toggle is very generic and we actually have actual toggling on this page :)

Reply:
@hugobarauna it is fine to comment it out and have it on the HTML. The only issue is the FAQ, which has fake questions and answers, so I would replace them by lorem ipsum and have it commented out.